app/views/categories/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<body>
+  <main>
+    <h2>Products by Category</h2>
+
+  <section class= "product-list">
+  <h4>Annual</h4>  
+    <% @product.categories each do |annual| %>   
```

Comment:
Did this display how you wanted? I think you need to change 'annual' to 'product' since the following code references product.name, product.description, etc.. Otherwise, it could be 'annual.name'.
```suggestion
    <% @product.categories each do |product| %>   
```

Reply:
For now that is what I want, I created a method on my product controller to self.find the category but I have to work on the logic to make it see many categories and I will update the views after that.